src/documentation/components/button/button.css

```diff
@@ -0,0 +1,67 @@
+@import "../../../index.css";
+
+.BUTTON{
+   
+    display: flex;
+    justify-content: space-between;
+}
+
+.btn {
```

Comment:
1. Class name on line 3 suggests that the name of class is going to be "button" or "BUTTON". However, here and everywhere else "btn" is the name used
2. The naming of class for the button component could have been more consistent